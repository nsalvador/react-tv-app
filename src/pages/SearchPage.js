import React from 'react';

import Layout from '../components/Layout';
import SearchPageHeader from '../components/SearchPageHeader';
import SearchPageContents from '../components/SearchPageContents';
import SearchProvider from '../provider/search';

const SearchPage = () => (
	<div>
		<Layout>
			<SearchProvider>
				<SearchPageHeader />
				<SearchPageContents />
			</SearchProvider>
		</Layout>
	</div>
);

export default SearchPage;
