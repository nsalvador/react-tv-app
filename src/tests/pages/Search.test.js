import React from 'react';
import { shallow } from 'enzyme';
import SearchPage from '../../pages/Search';

it('should render SearchPage correctly', () => {
	const wrapper = shallow(<SearchPage />);
	expect(wrapper).toMatchSnapshot();
});
