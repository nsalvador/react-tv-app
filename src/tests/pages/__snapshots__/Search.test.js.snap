// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SearchPage correctly 1`] = `
<div>
  <Layout>
    <ContextProvider
      value={
        Object {
          "dispatchError": [Function],
          "dispatchSearch": [Function],
          "error": "",
          "shows": Array [],
        }
      }
    >
      <SearchForm />
      <SearchList />
    </ContextProvider>
  </Layout>
</div>
`;
